src/forms/CalendarComponent.jsx

```diff
@@ -123,6 +123,11 @@ CalendarComponent.propTypes = {
 		PropTypes.element
 	]),
 	onChange: PropTypes.func, // provided by `redux-form`
+	required: PropTypes.bool,
+	requiredText: (props) => (
```

Comment:
This update will cause existing usage of these form components to fail propTypes in consumer apps. This may actually be a good thing if it only causes problems in dev, but a greater concern is that there is no fallback to using a `*`. 

I'd recommend either building in a fallback for the case of required fields without a `requiredText` prop, or filing a follow up bug to make usage updates in `mup-web` and `pro-web` with the upgrade of `meetup-web-components`.